### PR TITLE
IPV6 - Create Carp ipv6 error broadcast address

### PR DESCRIPTION
**Pull request by djgel**
Insert condicional ipv4 um broadcast test for new CARP address

### CODE REVIEW

**Comment by pfSense-git-updates** [comment]
Before this pull request can be accepted you must first sign a CLA as described at https://www.pfsense.org/about-pfsense/#cla. Please read for more details.

**Comment by djgel** [comment]
Tks...


Your subscription has been successfully processed. If appropriate, we will
email you shortly.
Order reference: *R9KFJ*

2017-02-06 14:30 GMT-02:00 pfSense commit bot <notifications@github.com>:

File: src/usr/local/www/firewall_virtual_ip_edit.php [hunk 1]
```diff
@@ -147,7 +147,7 @@ function find_last_used_vhid() {
 
 		if (isset($network_addr) && $_POST['subnet'] == $network_addr) {
 			$input_errors[] = gettext("The network address cannot be used for this VIP");
-		} else if (isset($broadcast_addr) && $_POST['subnet'] == $broadcast_addr) {
+		} else if (is_ipaddrv4($_POST['subnet']) && isset($broadcast_addr) && $_POST['subnet'] == $broadcast_addr) {
```
Comment (rbgarga) [inline src/usr/local/www/firewall_virtual_ip_edit.php]: The same result could be reached just removing line 145 and then $broadcast_addr will never be set for IPv6

**Comment by sbeaver-netgate** [comment]
Requested changed not addressed.

**Pull request closed without merging**